backend/mark_marker/src/main/scala/mark_marker/trainer/DatabaseConnector.scala

```diff
@@ -1,96 +1,55 @@
 package mark_marker.trainer
 
 import java.io._
-import java.sql.DriverManager
-import java.sql.Connection
-
-import edu.stanford.nlp.classify.{Classifier, ColumnDataClassifier}
+import java.sql.{Connection, DriverManager}
 
 import scala.io.Source
 
-class DatabaseConnector {
+class DatabaseConnector(dbName: String) {
   private val driver = "com.mysql.cj.jdbc.Driver"
-  private val url = "jdbc:mysql://tc.jstudios.ovh:3306/Automarker"
   private val username = "python"
-  private val psw = findPsw("dbpass.txt")
+  private val url = "jdbc:mysql://tc.jstudios.ovh:3306/Automarker"
+  private val psw = findPsw("../dbpass.txt")
```

Comment:
when a microservice is launched, the working dir is alabno
so it should be `dbpass.txt`